app/stac_api/models.py

```diff
@@ -599,7 +611,6 @@ class Language(models.TextChoices):
     proj_epsg = models.IntegerField(null=True, blank=True)
     title = models.CharField(max_length=255, null=True, blank=True)
     media_type = models.CharField(max_length=200)
-    href = models.URLField(max_length=255)
```

Comment:
probably got it wrong, but didn't we want to keep the 'href'-property?

Reply:
Not as model field.